### ISSUE TITLE
Missing outlines on masthead in High Contrast Mode mode. 

### ISSUE THREAD

**Issue by peterdobal** [carbon-design-system/carbon-for-ibm-dotcom#6306]
## Detailed description

> Describe in detail the issue you're having.

Setup needed for HCM:
Open IBM Docs in FF
Hit  Shift+Alt+PrtSc and start High Contrast Mode
After question  if you want to  enter to  HCM   wait  few sec and everything on view  should be change to  HCM(depends on  thinkpad older WS did not offer question and it took 1-3 seconds to load)
In some cases refresh of page is required.  

Bug

> Is this issue related to a specific component?
Masthead section

> What did you expect to happen? What happened instead? What would you like to
> see changed?

I would like to see  outlines on each component in masthead in HCM

> What browser are you working in?

FF 89.0, Windows 10

> What version of Carbon for IBM.com are you using?

> What offering/product do you work on? Any pressing ship or release dates we
> should be aware of?
IBMDOCS
## Steps to reproduce the issue

1. Run Precondition and start HCM
2. Open https://ibmdocs-dev.mybluemix.net/docs/en
3. With Tab navigate through whole masthead
4. ...

Actual  behavior:
User is not able to  see outlines in HCM in masthead section like IBM button,  Documentation & login  dropdown menu.

Expected behavior:
User should be able  see outlines for  navigation  on HCM in all  section of ibm docs.

## Additional information

- https://accessabilly.com/accessibility-issues-concerning-windows-high-contrast-mode/
- https://www.maxability.co.in/2021/06/13/windows-high-contrast-mode-and-accessibility-testing/
- https://johnsweetaccessibility.com/2020/06/focus-style-strategies/

**Comment by RobertaJHahn** [comment]
@peterdobal Could you post screen shots to this issue?  Thanks.

**Comment by peterdobal** [comment]
![Masthead HCM](https://user-images.githubusercontent.com/15247068/121523583-a8704580-c9f6-11eb-8882-dd95bef2e342.png)

**Comment by RobertaJHahn** [comment]
@Praveen-sr Can you have Ram look at this issue to determine the severity.  Thanks.

**Comment by RobertaJHahn** [comment]
Spoke to Ram in our Prod QA meeting this morning.  He said that WCAG does not require High Contrast Mode (HCM) testing.  He will talk to his testing team to find out what additional % time would be required if the HCM testing would be added to scope.  

The priority of this defect would be High if this fix is going to be done.

**Comment by ljcarot** [comment]
@peterdobal Due to our current accessibility budget, we have not included HCM testing this year as we've been focused on WCAG 2.1 level AA.  As a result, we will move the HCM issues to Backlog until next year.

**Comment by Stefan8412** [comment]
Hello, can you please give us update regards above mentioned isssue, thnanks
